conf/broker.conf

```diff
@@ -107,6 +107,9 @@ subscriptionRedeliveryTrackerEnabled=true
 # How frequently to proactively check and purge expired subscription
 subscriptionExpiryCheckIntervalInMinutes=5
 
+# Enable Key_Shared subscription (default is disabled)
+subscriptionKeySharedEnable=false
```

Comment:
I was just meaning to have it enabled by default (with an option to disable)

Reply:
Enabled it by default and add a unit test to ensure disable is work well.